README.md

```diff
@@ -1,18 +1,29 @@
 # Twilio Voice React Native Reference
 
-## What is this Project?
+This project consists of a backend server and a react native app demonstrating best practices for developing a Twilio Programmable Voice application.
 
-The Twilio Voice React Native Reference App is an example implementation of the Twilio Voice React Native SDK and serves to inspire developers who want to leverage the power of Twilio Programmable Voice in their React Native applications.
+## Features
 
-* See the Twilio Voice React Native SDK [here](https://github.com/twilio/twilio-voice-react-native).
+* Dialpad for making outgoing calls
+  * Call to a phone number
+  * Call to a Twilio Client<sup>*</sup>
+* Active call screen to interact with and control ongoing calls
+* Login flow for authenticating and authorizing users<sup>*</sup>
+* Registration and notifications for incoming calls<sup>*</sup>
 
-## Before you use this Project
+<sup>*</sup> Planned for a future release.
```

Comment:
I think it might make more sense to do a future release section?

Reply:
Good idea. Maybe something like "Coming soon". Let's see what the whole team thinks as well. @bobiechen-twilio @mhuynh5757

Reply:
Sounds good, I think a `Planned Features` or `Future Release` section is a good idea.